Makefile.PL

```diff
@@ -102,17 +102,30 @@ my %opts = (
 $opts{CAPI} = 'TRUE' if $Config{archname} =~ /-object\b/i;
 
 if (my $gccversion = $Config{gccversion}) {	# ask gcc to be more pedantic
-    warn "WARNING: Your GNU C $gccversion compiler is very old. Please upgrade it and rebuild perl.\n"
-	if $gccversion =~ m/^\D*(1|2\.[1-8])/;
-    print "Your perl was compiled with gcc (version $Config{gccversion}), okay.\n";
-    $gccversion =~ s/[^\d\.]//g; # just a number please
-    $opts{DEFINE} .= ' -W -Wall -Wpointer-arith -Wbad-function-cast';
-    $opts{DEFINE} .= ' -Wno-comment -Wno-sign-compare -Wno-cast-qual';
-    $opts{DEFINE} .= ' -Wmissing-noreturn -Wno-unused-parameter' if $gccversion ge "3.0";
-    if ($is_developer && $::opt_g) {
-        $opts{DEFINE} .= ' -DPERL_GCC_PEDANTIC -ansi -pedantic' if $gccversion ge "3.0";
-        $opts{DEFINE} .= ' -Wdisabled-optimization -Wformat'    if $gccversion ge "3.0";
-        $opts{DEFINE} .= ' -Wmissing-prototypes';
+    if ($gccversion =~ m/ Clang \d/) {
+      warn "WARNING: This is not GCC, but $gccversion. Not checking compiler version.\n";
```

Comment:
As perl itself builds fine with Clang, a warning like this seems out of place. Why not just
`print "Your perl was compiled with clang (version $1). As this is not gcc, version checking is skipped\n"`

Reply:
Yes indeed. Updated in https://github.com/Perl/perl5/commit/87e1a8bc7ef4ac3f732cb684d25918f19a909f68 but with `Clang` instead of `clang` and `GCC` instead of `gcc` and I added a `.`.